src/main/java/jav/Jav.java

```diff
@@ -0,0 +1,81 @@
+package jav;
+
+import java.util.Scanner;
+
+import jav.command.Command;
+import jav.exception.InvalidCommandException;
+import jav.exception.InvalidParamException;
+import jav.manager.FileManager;
+import jav.manager.ParserManager;
+import jav.manager.StorageManager;
+import jav.manager.UiManager;
+
+/**
+* The main program.
+*/
+public class Jav {
+    private static boolean isExiting = false;
+    public static void main(String[] args) {
+        // Print any neccessary UI upon starting up
+        String logo =
+                  "      ____.  _________   ____\n"
+                + "     |    | /  _  \\   \\ /   /\n"
+                + "     |    |/  /_\\  \\   Y   /\n"
+                + " /\\__|    /    |    \\     /\n"
+                + " \\________\\____|__  /\\___/\n"
+                + "                  \\/\n"
+                + "The Joy Amplifying Virtuoso!\n";
+                // Got ASCII Word Art from https://patorjk.com/software/taag/#p=display&f=Graffiti&t=Jav
+        System.out.println("Hello from\n" + logo);
+        System.out.println("<---------------------------------------------------------->");
+        UiManager.getInstance().printGreeting();
+        
+        // Load any neccessary data
+        Scanner scan = new Scanner(System.in);
+        try {
+            StorageManager.getInstance().load(FileManager.getInstance().loadStorageData());
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+
+        // Main program loop
+        while (!isExiting) {
```

Comment:
Nit: I think that using -ing form for boolean variables maybe a little contradictory since it's supposed to be only true or false but -ing refers to somewhere in between true and false. Perhaps you can consider variable names like isComplete, isFinished, etc.